app/HomeScreen.jsx

```
@@ -1,487 +1,550 @@
+        }),
+      ])
+    ).start();
+  }, []);
+  useEffect(() => {
+    Promise.all([axiosInstance.get("/category"), axiosInstance.get("/images")])
+      .then(([catRes, imgRes]) => {
+        const images = imgRes.data;
+        let cats = catRes.data.slice(0, 7).map((cat, idx) => ({
+          key: cat._id,
+          label: cat.name,
+          icon: categoryImages[idx] || require("../assets/images/cpu.png"),
```

ESLint

[error] 'categoryImages' is not defined.

(no-undef)